### PR TITLE
LIME-332: Change Sign Up button to Sign In

### LINKED ISSUE #332: Change Sign Up button to Sign In in the header of Landing page
Image

Change label of button with it's redirection to Sign In to improve UX

### PR DESCRIPTION
**Pull request by VasylynaKolodiy**

### CODE REVIEW

File: frontend/src/bundles/landing/pages/landing.tsx [hunk 1]
```diff
@@ -51,8 +51,8 @@ const Landing = (): JSX.Element => {
         }
     }, [dispatch, user]);
 
-    const signUpHandler = useCallback((): void => {
-        dispatch(appActions.navigate(AppRoute.SIGN_UP));
+    const signInHandler = useCallback((): void => {
```
Comment (dimapopovych) [inline frontend/src/bundles/landing/pages/landing.tsx]: naming should start with handle by quality criteria

**Review by dimapopovych — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/landing/pages/landing.tsx (hunk 1 above)
Comment (VasylynaKolodiy) [reply frontend/src/bundles/landing/pages/landing.tsx]: Thanks, fixed it

**Review by valery-chumak — DISMISSED** [review]

**Review by stpalamar — DISMISSED** [review]

File: frontend/src/bundles/landing/pages/landing.tsx [hunk 2]
```diff
@@ -105,10 +105,10 @@ const Landing = (): JSX.Element => {
                     </nav>
                     <Button
                         size={ComponentSize.MEDIUM}
-                        label="Sign Up"
+                        label="Sign In"
                         variant={ButtonVariant.PRIMARY}
                         className="basis-40"
-                        onClick={signUpHandler}
+                        onClick={handleSignIn}
```
Comment (nikita-remeslov) [inline frontend/src/bundles/landing/pages/landing.tsx:115]: we just need to update 1 button!

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/landing/pages/landing.tsx (hunk 2 above)
Comment (VasylynaKolodiy) [reply frontend/src/bundles/landing/pages/landing.tsx:115]: I fixed it, thanks

**Review by dimapopovych — APPROVED** [review]

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**